### DISCUSSION TITLE
Change 80:5000 port mapping for a noob?

### DISCUSSION THREAD

**Discussion by CrouchingWorm** [robmarkcole/HASS-Deepstack#274] [Q&A]
Hi all,
I've been running Deepstack for a while on a different machine to that of my HA server.
The subscription has since expired on that machine.
So have decided to use this add-on on my HA machine.
Thing is I already have Frigate running on that machine which also uses port 5000.

My question:
How do I change the port of Deepstack for this container/addon from 80:5000 -> 80:5001 ?
Is there a config file somewhere? a CLI command or reference.
I'm assuming by running the add-on it's actually executed the docker command on my host machine?

*update* - I can see the the add-on hasn't executed the docker command.
All good. :) I can figure it out from here.
ALL SORTED. I can't find where to delete this post.. SORRY!

**Comment by robmarkcole** [comment]
Solved
